src/sx/src/__tests__/injectRuntimeStyles.test.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import { render } from '@testing-library/react';
+
+import * as sx from '../../index';
+
+it('injects runtime styles', () => {
+  render(<style data-adeira-sx />);
+
+  const styleTag = globalThis.document.querySelector('style');
+  expect(styleTag?.sheet?.cssRules).toHaveLength(0);
+
+  sx.create({
+    test: {
+      opacity: 0,
+      height: '100px',
+    },
+  });
+
+  expect(styleTag?.sheet?.cssRules).toHaveLength(2);
+  expect(styleTag.sheet.cssRules).toMatchInlineSnapshot(`
```

Comment:
Why are the optional chains required on line 20 and not on line 21? 🤔

Reply:
Yeah, good question, I didn't notice. 

it seems like it was not necessary after adding `globalThis.document`, so I guess that makes the styleTag any?

I will clean it up.

Reply:
Might be. `globalThis` and `global` are type of `any`. Sometimes it's ok to access these global objects directly and just explain it to Eslint like so: `/* global document */`

Reply:
Yeah, that also works 🤔 

My idea was showing lint-error even if I added global document, but I see it passes in the terminal.